sigstore/verify/models.py

```diff
@@ -438,3 +451,67 @@ def rekor_entry(self, client: RekorClient) -> LogEntry:
             raise InvalidRekorEntry
 
         return entry
+
+    def to_bundle(self) -> Bundle:
+        """Converts VerificationMaterials into a Bundle. Requires that
+        the VerificationMaterials have a Rekor entry loaded. This is
+        the reverse operation of VerificationMaterials.from_bundle()
+        """
+        if not self.has_rekor_entry:
+            raise InvalidMaterials(
+                "Must have Rekor entry before converting to a Bundle"
+            )
+        rekor_entry = self._rekor_entry
+        assert rekor_entry is not None
```

Comment:
`bandit` doesn't like `assert`, maybe we can make this a `type: ignore` with an explicit annotation to make `mypy` happy?

Reply:
Done!